.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: python
-
 service:
   - docker
-
 before_install:
-- docker build . -t todo-app:dev --target=development
-- docker run -d -p 5000:5000 todo-app:dev --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:dev 
+   - docker build . -t todo-app:test --target=test 
+   - docker run --name travisImage -d -p 5000:5000 -e key -e token --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:test 
+   - sleep 5
+   - docker ps -a
```

Comment:
Is there a reason to put this in the `before_install` step?
Since it's necessary only for running the tests as in the script, I'd be inclined to keep it close by for simplicity

Also, the `docker ps -a` command was presumably added for debugging and should now be something we can get rid of, can we also remove the sleep command? I know you've said you prefer entrypoints for background commands/web servers, but this is a good example of why they can be helpful - the entrypoint is naturally linked to the lifecycle of the container, and we don't need to worry about sleeping!